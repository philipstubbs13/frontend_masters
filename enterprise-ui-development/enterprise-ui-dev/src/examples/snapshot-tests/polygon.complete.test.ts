import { describe, it, expect } from 'vitest';
import { Polygon } from '$lib/polygon';

describe('Polygon', () => {
  it('should match the snapshot', () => {
    const polygon = new Polygon(8, 10);
    expect(polygon.toJSON()).toMatchSnapshot();
  });
});
