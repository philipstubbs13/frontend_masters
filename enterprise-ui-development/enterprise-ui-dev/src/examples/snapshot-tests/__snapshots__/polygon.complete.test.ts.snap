// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Polygon > should match the snapshot 1`] = `
{
  "area": 482.842712474619,
  "lengthOfSides": 10,
  "perimeter": 80,
  "sides": 8,
  "sumOfAngles": 1080,
}
`;
